python/cuml/feature_extraction/_stop_words.py

```diff
@@ -0,0 +1,46 @@
+# This list of English stop words is taken from the "Glasgow Information
+# Retrieval Group". The original list can be found at
+# http://ir.dcs.gla.ac.uk/resources/linguistic_utils/stop_words
```

Comment:
Maybe add a little comment to note that this is also matching sklearn stop_words and cite their license here too... even though it's derived from Glasgow, both should get credit.

Reply:
Agreed